src/io/flutter/run/daemon/DaemonJsonInputFilterProvider.java

```diff
@@ -31,9 +31,14 @@
     public List<Pair<String, ConsoleViewContentType>> applyFilter(String text, ConsoleViewContentType contentType) {
       if (!VERBOSE) {
```

Comment:
It's just for use during development - when you're working on the launch code itself.